tests/infrastructure/utils.py

```diff
@@ -0,0 +1,12 @@
+from typing import Any, List
+
+import bitmath
+
+
+def get_hugepages_gib_values(workers: List[Any]) -> List[int]:
```

Comment:
you use it in multiple places, better to make it as a session scope fixture
or at least add caching

Reply:
get_hugepages_gib_values is called twice and at a session scope level(xfail_if_no_huge_pages and hugepages_gib_max) so it will be calculated twice.which is kind of ok'ish. 
Best solution would be to make it afixture so that it will be calculated once in session. what do you think. cache would be overhead in this case